cmd/update/cluster/flag.go

```diff
@@ -0,0 +1,61 @@
+package cluster
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const (
+	flagName           = "name"
+	flagReleaseVersion = "release-version"
+	flagScheduledTime  = "scheduled-time"
+	flagProvider       = "provider"
+)
+
+type flag struct {
+	config         genericclioptions.RESTClientGetter
+	print          *genericclioptions.PrintFlags
+	Name           string
+	ReleaseVersion string
+	ScheduledTime  string
+	Provider       string
+}
+
+func (f *flag) Init(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&f.Name, flagName, "", "Name of the cluster to update.")
+	_ = cmd.Flags().MarkHidden(flagName)
```

Comment:
Since all these flags are hidden, am I correct to assume that this command is mainly for GS staff to use?
This is a very useful command. 
What do you think about also supporting the "instant" case of just updating the label without `--scheduled-time` like we have in happa? Or do you think we should rather encourage scheduling?

Reply:
Make sense, so shall `--scheduled-time` be optional and without specifying we update the release version directly?

Reply:
If it's not too much effort to include, I think it would be nice. :)

Reply:
Ok done 🙂 , PTAL